addons/tagging/initSettings.sqf

```diff
@@ -0,0 +1,11 @@
+
+if (EGVAR(common,aceTagging)) then {
+    [
+        QGVAR(enabled),
+        "CHECKBOX",
+        [LSTRING(Enabled), LSTRING(Enabled_Description)],
+        LSTRING(DisplayName),
+        true,
```

Comment:
I'd rather had all our custom features opt-in instead of opt-out if possible. Our server settings will auto enable it anyway. But your call.